javascript/core/objects/keywords-in-hash-keys.md

```diff
@@ -24,7 +24,9 @@ notes: 'Same issue, not sure if it''s not really really basic.'
 
 aspects:
   - introduction
-
+  - deep
```

Comment:
I would remove `deep`, it's definitely a misc. detail but it's not a complexity and doesn't require two previous insights to understand.